### ISSUE TITLE
WCAG20-TECHS ARIAパート コメント(5) - ARIA4・ARIA5 「ユーザーインターフェースコンポーネントのロール」をより具体的に説明すべき

### ISSUE THREAD

**Issue by momdo** [momdo/momdo.github.io#17]
たとえばARIA4の現行訳（http://momdo.github.io/WCAG20-TECHS_aria.html#ARIA4 ）の「解説」のセクションによると、

> このテクニックの目的は、ロールのWAI-ARIAの定義で定義されたいずれかの非抽象の値を持つrole属性を用いて要素のロールを定義することです。WAI-ARIA仕様は、どのようにロールが他のロール、そして各ロールのステートおよびプロパティーと関係するのか、各ロールの有益な説明を提供します。リッチインターネットアプリケーションロールが新しいユーザーインターフェースウィジェットを定義する場合、ロールの公開はユーザーがウィジェットおよびウィジェットと対話する方法を理解できるようにします。

とある（引用部分は原文の<a href="https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/ARIA4.html">Description</a>のセクションに相当する）のだけれども、現在のこの説明だと「ARIA4：ユーザーインターフェースコンポーネントのロールを公開するWAI-ARIAロールを使用する」というテクニックで、あたかも全てのロールが指定できるかのような印象を受ける、というのが指摘のポイント。

おそらく趣旨としては、ARIA4テクニックで指定できるロールは、WAI-ARIA仕様の<a href="http://momdo.github.io/wai-aria/roles.html#role_definitions">5.3.2. ウィジェットロール</a>に限定するのが妥当ではないか、と。また、ARIA in HTML仕様により、列挙できる組み合わせは限られるはずなので、より踏み込んだ説明ができるのではないか（またそういう説明にしないと、このテクニックの意味がないと思う）。

ARIA5についても同様。

**Comment by momdo** [comment]
issue案：

ARIA4/5: more clarify a user interface comportnent

In description section of ARIA4, for instance, the draft says "The objective of this technique is to define the role of an element using the role attribute with one of the non-abstract values defined in the WAI-ARIA Definition of Roles." I think the scope of the role of this description is too wide. In my view, it may be appropriate to limit <a href="http://www.w3.org/TR/wai-aria/roles#widget_roles_header">WAI-ARIA Widget Roles</a>. Additionally, according to the <a href="https://www.w3.org/TR/html-aria/">ARIA in HTML</a> spec, combinations which can be specified role (or state) are limited.

**Comment by momdo** [comment]
ARIA4/5: Clarification of a user interface component

In description section of ARIA4, the draft says "The objective of this technique is to define the role of an element using the role attribute with one of the non-abstract values defined in the WAI-ARIA Definition of Roles." I think the scope of the role of this description is too wide. In my view, it should be limited to WAI-ARIA Widget Roles. Additionally, according to the ARIA in HTML spec, possible combination of roles and states are limited thus it should be noted as well.

**Comment by momdo** [comment]
issue 150として報告した。

**Comment by momdo** [comment]
一旦閉じる
